internal/comparetest/options.go

```diff
@@ -33,6 +34,11 @@ type LogsCompareOption interface {
 	applyOnLogs(expected, actual plog.Logs)
 }
 
+// TracesCompareOption can be used to mutate expected and/or actual traces before comparing.
+type TracesCompareOption interface {
```

Comment:
Can you please add traces support in `IgnoreResourceAttributeValue` option as a follow-up PR?

Reply:
I will create a pr to record the expected enhancements of traces. @dmitryax